### PR TITLE
Add toast, rating and list components

### PR DESCRIPTION
**Pull request by jalajcodes**
Just check `.scss` files.
 `.css` and `.map` files are automatically generated and don't need to be reviewed.

### CODE REVIEW

**Review by vrinda-mahajan — APPROVED** [review]
Good to merge.👍

File: components/toast/toast.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  p {
+    margin: 1rem 0.7rem 1rem 0;
+  }
+  &__icon {
+    display: inline-block;
+    margin: 0 0.8rem;
+  }
+  &--primary {
+    background-color: var(--primary);
+    color: var(--white);
```
Comment (anik31) [inline components/toast/toast.scss:20]: add space after each ruleset

**Review by anik31 — APPROVED** [review]
Everything looks good, you can merge.

File: components/toast/toast.scss (hunk 1 above)
Comment (jalajcodes) [reply components/toast/toast.scss:20]: Thanks will do that.

**Pull request merged**